Tasks/CMake/cmaketask.js

```diff
@@ -0,0 +1,17 @@
+/// <reference path="../../definitions/vso-task-lib.d.ts" />
```

Comment:
We shouldn't checkin the .js file right?  Just the ts and the build generates the js ...